src/pcp-reboot-init/pcp-reboot-init.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh
+#
+# Copyright (c) 2024 Ken McDonell.  All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+# Intended as "run after reboot" to make sure transient things that PCP
+# needs have been setup ...
+#
+
+tmp=/var/tmp/pcp-reboot-init-$$
```

Comment:
Should this use mktemp here like most PCP scripts do?  Think so.

Reply:
Yep, fixed in commit 35600595c